src/models/continuity/ContGateTurn.ts

```diff
@@ -30,17 +36,69 @@ export default class ContGateTurn
 
   getHumanReadableText(): string {
     if (this.humanReadableText !== "") return this.humanReadableText;
-    // TODO: Implement
-    return "";
+    const rotation: string = Math.sign(this.angle) === 1 ? "CW" : "CCW";
+    return `GT${this.marchType} ${this.duration} COUNTS ${Math.abs(
+      this.angle
+    )} DEGREES ${rotation}`;
   }
 
-  /* eslint-disable @typescript-eslint/no-unused-vars */
-  addToFlow(
-    flow: FlowBeat[],
-    startDot: StuntSheetDot,
-    endDot?: StuntSheetDot
-  ): void {
-    // TODO: Implement
+  addToFlow(flow: FlowBeat[], startDot: StuntSheetDot): void {
+    const [startx, starty]: [number, number] = startPositionHelper(
+      flow,
+      startDot
+    );
+    let centerPoint: [number, number] | undefined = this.centerPoints.get(
+      startDot.id
+    );
+    if (!centerPoint) {
+      centerPoint = [0, 0];
```

Comment:
Here I have it defaulting to [0, 0] as the center point if a dot hasn't been assigned a center yet. Thoughts on this?

Reply:
This is fine for now, sure. Maybe create an issue to show a message or something to indicate that he user needs to set the center of the gate turn.

Reply:
This is fine, but a little comical. Classic CalChart's default would be 'directly under their current position'. Whatever you decide here is good with me. @MichaelTamaki 's comment is a really valuable one -- we should probably tell the user somehow that things are misconfigured.